Issue 05

disandroa opened the pull request:
# Description

Created scripts to query data from derived.tax_year_assessment_bins and exported to JSON file in the "public" data bucket under the configs/ folder as current_assessment_bins.json.

Resolves #\[issue 5\]

## Type of change

- [x] New feature
- [ ] Bug fix
- [ ] Small fix/change
- [ ] Documentation

## What should the reviewer know?


## Post-merge follow-ups

- [x] No action required
- [ ] Actions required (specified below)

disandroa committed:
created initial files for issue 5

disandroa committed:
updated code in issue 5 folder

disandroa committed:
deployed and ran function generate_assessment_chart_configs

The cloud function associated with issue 5 (generate_assessment_chart_configs) has been successfully deployed and ran. Now, there exists a JSON file with the information from the derived.tax_year_assessment_bins table

junyi2022 commented on tasks/5_tax_year_assessments_config/main.py:
```diff
@@ -0,0 +1,41 @@
+    dataset_ref = bigquery.DatasetReference(project, dataset_id)
+    table_ref = dataset_ref.table(table_name)
+    job_config = bigquery.job.ExtractJobConfig()
+    job_config.destination_format = bigquery.DestinationFormat.NEWLINE_DELIMITED_JSON
```
**Suggestion:** The file format that you are exporting here is JSONL, which the JavaScript for the frontend tasks cannot read directly.  It is possible to find some libraries that can help you parse JSONL in JavaScript, but it may be better to change it to actual JSON before you store it in the cloud storage. On the other hand, the default file type is CSV, and you can just export the CSV to cloud storage. See documentation [here](https://cloud.google.com/bigquery/docs/samples/bigquery-extract-table). However, JavaScript also needs libraries to parse CSV, but the number of possible libraries you can choose from will be more than the libraries that can deal with JSONL.

Either JSONL, JSON, or CSV can work on the front end, but it is a decision that you'd better consider/coordinate with the front-end team member to provide smoother collaboration.

junyi2022 reviewed (commented):
I have left a comment below, which is about the file type you are going to export to cloud storage.

disandroa committed:
using new code for main.py and updated issue5-terminal-code.sh and requirements.txt

solanoamy committed:
Updated function to create .json rather than .jsonl

Adapted code to create geojson into workflow for extracting json from BigQuery table. Changed data types to strings for export
